ckanext/datarequests/db.py

```diff
@@ -164,14 +174,31 @@ def get_datarequest_followers_number(cls, **kw):
 
         # FIXME: References to the other tables...
         followers_table = sa.Table('datarequests_followers', model.meta.metadata,
-            sa.Column('id', sa.types.UnicodeText, primary_key=True, default=uuid4),
-            sa.Column('user_id', sa.types.UnicodeText, primary_key=False, default=u''),
-            sa.Column('datarequest_id', sa.types.UnicodeText, primary_key=True, default=uuid4),
-            sa.Column('time', sa.types.DateTime, primary_key=True, default=u'')
-        )
+                                   sa.Column('id', sa.types.UnicodeText, primary_key=True, default=uuid4),
+                                   sa.Column('user_id', sa.types.UnicodeText, primary_key=False, default=u''),
+                                   sa.Column('datarequest_id', sa.types.UnicodeText, primary_key=True, default=uuid4),
+                                   sa.Column('time', sa.types.DateTime, primary_key=True, default=u'')
+                                   )
 
         # Create the table only if it does not exist
         followers_table.create(checkfirst=True)
 
         model.meta.mapper(DataRequestFollower, followers_table,)
 
+
+def update_db(model):
+    '''    
+    A place to make any datarequest table updates via SQL commands
+    This is required because adding new columns to sqlalchemy metadata will not get created if the table already exists
+    '''
+    meta = MetaData(bind=model.Session.get_bind(), reflect=True)
+
+    # Check to see if columns exists and create them if they do not exists
+    if closing_circumstances_enabled:
+        if 'datarequests' in meta.tables and 'close_circumstance' not in meta.tables['datarequests'].columns:
+            log.info("DataRequests-UpdateDB: 'close_circumstance' field does not exist, adding...")
+            DDL('ALTER TABLE "datarequests" ADD COLUMN "close_circumstance" varchar(100) NULL').execute(model.Session.get_bind())
```

Comment:
we have a problem, in constants we have ```CLOSE_CIRCUMSTANCE_MAX_LENGTH = 255``` but we set database table to only hold 100.